src/frontend/pages/recruitment/Configuration/ConfigurationPage.js

```diff
@@ -86,23 +90,101 @@ const TimeContainer = styled.div`
 `;
 
 const ConfigurationPage = () => {
-  const [firstDeadline, setFirstDeadline] = React.useState(new Date());
-  const [secondDeadline, setSecondDeadline] = React.useState(new Date());
-  const [orientationDate, setOrientationDate] = React.useState(new Date());
-  const [confirmDeadline, setConfirmDeadline] = React.useState(new Date());
-  const [startTime, setStartTime] = React.useState(new Date());
-  const [endTime, setEndTime] = React.useState(new Date());
+  const { configuration, updateConfiguration } = useConfiguration();
+
+  const [config, setConfig] = useState({
```

Comment:
It's a bit confusing since `config` is so similar in name to `configuration`.

Could any of these be renamed ?